<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: #ffffff;
}
QLineEdit{
background-color: #5EBDB2;
border-radius: 5px;
color: #ffffff;
}
QListWidget{
background-color: #c8e5e3;
border-radius: 10px;
}
QLabel{
color: #142b3b;
}
QGroupBox{
color: #1792a4;
border-radius: 10px;
background-color: #e8e8e7;
}
QPushButton{
 	background-color: #142b3b;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 13px;
    min-width: 7em;
    padding: 3px;
	color:#c8e5e3;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>&quot;Call Me&quot;, (Maybe)?</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lista de clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Guardar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cargar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>&quot;Call Me&quot;, (Maybe)?</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Registrar clientes</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Datos personales</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">MainWindow-&gt;setStyleSheet(&quot;QLineEdit { background-color: yellow }&quot;);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Dirección</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Localidad</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Provincia</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>N.I.F</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Fecha de 
 nacimiento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">myWidget = new QWidget(this);
myWidget-&gt;setGeometry(0,0,300,100);
myWidget-&gt;setStyleSheet(&quot;background-color:black;&quot;);
myWidget-&gt;show();</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Datos inscripción</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>&quot;Call Me&quot;, (Maybe)?</family>
      </font>
     </property>
     <property name="text">
      <string>Días límite</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>301</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Grande</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>                Id - Nombre - Apellidos</string>
    </property>
    <widget class="QListWidget" name="lista">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>301</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_Crear_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear tarjeta</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>20</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>&quot;Call Me&quot;, (Maybe)?</family>
    </font>
   </property>
   <widget class="QMenu" name="menuRegistro">
    <property name="title">
     <string>Registro</string>
    </property>
    <addaction name="actionRegistrar"/>
   </widget>
   <widget class="QMenu" name="menuClientes">
    <property name="title">
     <string>Clientes</string>
    </property>
    <addaction name="recibos"/>
    <addaction name="consultar"/>
   </widget>
   <widget class="QMenu" name="menuInventario">
    <property name="title">
     <string>Inventario</string>
    </property>
    <addaction name="inventario"/>
   </widget>
   <addaction name="menuRegistro"/>
   <addaction name="menuClientes"/>
   <addaction name="menuInventario"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>&quot;Call Me&quot;, (Maybe)?</family>
    </font>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCrear_tarjeta">
   <property name="text">
    <string>Crear tarjeta</string>
   </property>
   <property name="font">
    <font>
     <family>&quot;Call Me&quot;, (Maybe)?</family>
    </font>
   </property>
  </action>
  <action name="actionFacturas">
   <property name="text">
    <string>Lista mes</string>
   </property>
  </action>
  <action name="recibos">
   <property name="text">
    <string>Cobros y facturas</string>
   </property>
  </action>
  <action name="consultar">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
  <action name="actionRegistrar">
   <property name="text">
    <string>Registrar</string>
   </property>
  </action>
  <action name="inventario">
   <property name="text">
    <string>Consultar inventario</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
